torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -2760,6 +2842,11 @@ def _post_backward_hook(self, param: Parameter, *unused: Any) -> None:
                 # shard when rebuilding the full params in the pre_beckward_hook.
                 self._free_mp_shard(cast(List[FlatParameter], [param]))
 
+            if (self.use_param_exec_order_policy() and self._param_exec_order_prep_stage):
+                # In self._fsdp_params_exec_order, the parameters are ordered based on
```

Comment:
specify that this is the order in the first iteration?

Reply:
Done, thanks!